concepts/objects/about.md

```diff
@@ -93,7 +93,7 @@ A couple of frequent cases for object construction have short-hand syntax.
 
 ## Indexing
 
-Values are retrieved from an object with _dot notation_.
+Values are retrieved from an object with "dot notation".
```

Comment:
The italicized text also appears in the introcution.md in concepts/ and in exercises/concept/

Reply:
Sorry. I didn't mean to include this. I brought this up on the forum for discussion, as this isn't a small change.
Reverted this line.